plugins/inputs/kube_inventory/kube_state.go

```diff
@@ -10,7 +10,7 @@ import (
 	"sync"
 	"time"
 
-	"github.com/kubernetes/apimachinery/pkg/api/resource"
+	"k8s.io/apimachinery/pkg/api/resource"
```

Comment:
github.com/kubernetes and k8s.io are mirrors of each other.  In order to not end up with two versions in go.mod and golang type conflicts in the future we need to use one mirror in telegraf.  The prometheus plugin uses github.com/kubernetes/apimachinery, so it makes sense to use github.com/kubernetes everywhere in telegraf and remove all references to k8s.io.